src/main/resources/assets/trafficcontrol/misc/signs.json

```diff
@@ -241,11 +242,10 @@
     "variant": "39"
   },
   {
-    "name": "Toll Ahead",
```

Comment:
I don't agree with making this change.  If a player has a map where they used a ton of 40s instead of 37s, then it's going to require them to go through and update all of those signs for no real good reason.  The problem is Minecraft doesn't have a good "migration" system where we can just change these for the player so until we can figure that out, I don't think we can do this.